WordPress/Classes/Utility/Logging/SentryStartupEvent.swift

```diff
@@ -6,10 +6,19 @@ WARNING: This class was created to track events of failures during
 startup time. This will block the thread. Do not use unless you're sure.
 */
 @objc class SentryStartupEvent: NSObject {
-    private var errors = [String]()
+    private typealias UserInfo = [String: Any]
+    private typealias ErrorWithInfo = (String, UserInfo)
+
+    private var errors = [ErrorWithInfo]()
 
     func add(error: NSError, file: String = #file, function: String = #function, line: UInt = #line) {
-        errors.append("\(function) (\(file):\(line)) \(error.localizedDescription) | userInfo: \(error.userInfo)")
+        let filename = (file as NSString).lastPathComponent
+        let info: UserInfo = [
+            "Description": error.description,   // Intentionally not using `localizedDescription`, as it helps to have error in Sentry in a single language
```

Comment:
To clarify, `description` contains a description of the error object. Something like this:

> Error Domain=NSURLErrorDomain Code=-1004 "No se ha podido establecer conexión con el servidor." UserInfo={NSUnderlyingError=0x2832203f0 {Error Domain=kCFErrorDomainCFNetwork Code=-1004 "(null)" UserInfo={_kCFStreamErrorCodeKey=61, _kCFStreamErrorDomainKey=1}}, NSErrorFailingURLStringKey=http://192.168.1.38:8081/status, NSErrorFailingURLKey=http://192.168.1.38:8081/status, _kCFStreamErrorDomainKey=1, _kCFStreamErrorCodeKey=61, NSLocalizedDescription=No se ha podido establecer conexión con el servidor.}

It's not an English version of the `localizedDescription` method, and the human-readable messages in there will still be localized.

I suggested the other code snippet trying to avoid changing the current code too much, but if we're going with this solution, I'd probably store the error directly here instead of the `UserInfo` type.

Perhaps instead of `ErrorWithInfo` we can store an array of `ErrorWithCaller = (Error, String)`, or make it a proper struct so the enumeration code looks a bit cleaner.